<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_MainWindow</class>
 <widget class="QMainWindow" name="Ui_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1" rowspan="3">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QListView" name="listView">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="accessibleDescription">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="movement">
           <enum>QListView::Static</enum>
          </property>
          <property name="resizeMode">
           <enum>QListView::Fixed</enum>
          </property>
          <property name="layoutMode">
           <enum>QListView::SinglePass</enum>
          </property>
          <property name="modelColumn">
           <number>0</number>
          </property>
          <property name="uniformItemSizes">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 14pt;
background: #0F7FFF;
color: white;
border-radius: 8px;</string>
            </property>
            <property name="text">
             <string>Сохранить</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 14pt;
background: #0F7FFF;
color: white;
border-radius: 8px;</string>
            </property>
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 14pt;
background: #0F7FFF;
color: white;
border-radius: 8px;</string>
            </property>
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="Scheduler" name="calendarWidget">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="gridVisible">
         <bool>true</bool>
        </property>
        <property name="horizontalHeaderFormat">
         <enum>QCalendarWidget::ShortDayNames</enum>
        </property>
        <property name="verticalHeaderFormat">
         <enum>QCalendarWidget::NoVerticalHeader</enum>
        </property>
        <property name="navigationBarVisible">
         <bool>true</bool>
        </property>
        <property name="dateEditEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 24pt;
background: #0F7FFF;
color: white;
border-radius: 8px;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Планировщик&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Scheduler</class>
   <extends>QCalendarWidget</extends>
   <header location="global">Scheduler.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
